bounded-collections/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [0.1.6] - 2023-05-05
+- Added `serde` feature, which can be enabled for no `std` deployments.
```

Comment:
nit: If we are following semver here, should this technically be a minor version bump? Since we are adding functionality.

Reply:
https://github.com/paritytech/parity-common/pull/745/commits/02f09de224499ab9b5dc73c6ef324660f9b5c097

Reply:
Thanks, looks like we weren't very strict about semver in the past, but good to start now. :)

Reply:
Actually, since we're still at `0.y.z` we might have been following the common scheme where `y` is reserved for breaking changes.

The [spec](https://github.com/semver/semver/blob/master/semver.md) recommends always bumping `y` on new releases when `x=0`, but probably we shouldn't change the strategy we were already following. Maybe we should revert? 🙈

Reply:
Both are fine for me. But following previous strategy makes sense. I'll revert.